trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340/partition/flash_layout.h

```diff
@@ -40,7 +41,8 @@
  * 0x000f_0000 Protected Storage Area (16 KB)
  * 0x000f_4000 Internal Trusted Storage Area (8 KB)
  * 0x000f_6000 NV counters area (4 KB)
- * 0x000f_7000 Unused
+ * 0x000f_7000 Unused (4 KB)
+ * 0x000f_8000 Non-secure storage (32 KB)
```

Comment:
Should we state here that the presence of this partition is conditional? Especially since the option is default `OFF`.

Reply:
Yes, good idea.